conf/messages

```diff
@@ -79,11 +79,15 @@ honestyDeclaration.heading = Honesty declaration
 honestyDeclaration.p1 = I confirm that:
 honestyDeclaration.li.1.1 = because {0},
 honestyDeclaration.li.1.2 = I was unable to submit the VAT Return due on {0}
+agent.honestyDeclaration.li.1.2 = they were unable to submit the VAT Return due on {0}
 honestyDeclaration.li.2 = no one else was available to make the submission for me
+agent.honestyDeclaration.li.2 = no one else was available to make the submission for them
```

Comment:
How are testing this message (or similar ones) like we have tested for messages in case of agent and VAT trader for 'crime'.

Reply:
Please ignore, I think it's done in 'li2AgentText'